### PR TITLE
keran4 adds Mountain View hometown file

### PR DESCRIPTION
**Pull request by keran4**
This PR will add content for the Mountain View hometown file :mount_fuji: 

Some open :question: before I merge:

1. Have been to Mountain View, if so, what did you see in there?
1. What did you enjoy
1. What did you see
1. Did you checkout the Google office, did you happen to see the google self driving car?

### CODE REVIEW

File: keran4-MounainView.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Mountain View, CA
+
+### Restaurants
+- Sakoon
+- Shiva’s
+
+### Local Attractions
+- SFO downtown :)
```
Comment (githubteacher) [inline keran4-MounainView.md]: @keran4 I have been to SFO but not Mountain View -- sounds 😎

Comment (akhil2000) [inline keran4-MounainView.md]: SFO Downtown is nice..

**Comment by rajavineel** [comment]
Can you view my comment?

**Comment by keran4** [comment]
@hkparekh I addressed in commit id:  85876dd

**Comment by msowrierajan** [comment]
:+1:

**Pull request merged**